pino.d.ts

```diff
@@ -677,9 +677,19 @@ declare namespace P {
     type LogDescriptor = Record<string, any>; // TODO replace `any` with `unknown` when TypeScript version >= 3.0
 
     interface Bindings {
+        [key: string]: any;
+    }
+
+    interface ChildLoggerOptions {
         level?: Level | string;
         serializers?: { [key: string]: SerializerFn };
-        [key: string]: any;
+        customLevels?: { [key: string]: number };
```

Comment:
I do not know if it is intended not to include both `customLevels` and `formatters`, as it do not support in `broswer`.